src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3815,6 +3818,123 @@
     public static final int TEXT_BLOCK_LITERAL_END =
             GeneratedJavaTokenTypes.TEXT_BLOCK_LITERAL_END;
 
+    /**
+     * The {@code yield} keyword.  This element appears
+     * as part of a yield statement.
+     *
+     * <p>For example:</p>
+     * <pre>
+     * return switch (mode) {
```

Comment:
```
int yield = 0;
return switch (mode) {
```

to show that yield is not literal out of switch.

Reply:
Fixed.